### DISCUSSION TITLE
Collapse Parent Row

### DISCUSSION THREAD

**Discussion by philiprenich** [vincode-io/Zavala#104] [Ideas]
If I'm working on sub-points in an outline, I would like to be able to collapse the parent they are underneath so that I can easily close that row and ignore it, moving on to the next piece of work. I thought `Collapse all in row` would do this if the current row didn't have further nesting, but it's just for the current row and its children. 

Is this something that would be useful to others?

**Comment by vincode-io** [comment]
So you are thinking that we would have an option like "Collapse Parent Row" that could be used on rows without current nesting? Would it always collapse the top most parent and everything in it or just the top most parent itself?

**Reply by philiprenich** [reply]
Probably easiest to diagram this a bit more.

- Top level idea 1.
  - A second layer 1.a
    - A third layer 1.a.i
    - Another third layer 1.a.ii *<-- Cursor is on this row*
    - A third third layer 1.a.iii
  - Another second layer 1.b
- Another top level idea 2.
  - A second layer 2.a

I would like to say "Collapse Parent Row" and have `1.a` collapse. This would be useful as I've "completed" writing / organising 1.a and want to de-clutter as I continue outlining. I'd like to do that w/out having to arrow up several rows or move to the mouse to do so.

**Reply by vincode-io** [reply]
I've added the ability to collapse the parent row when in a child row. This will be in the upcoming 1.1 release.
